README.md

```diff
@@ -25,28 +82,218 @@ This repository contains extension code for SQL Server input and output bindings
+    ```
+
+- Congratulations! You have successfully created an Azure SQL Database! Make sure you complete [Quick Start](#Quick-Start) before continuing to the rest of the tutorial.
+
+### Input Binding Tutorial ###
+
+Note: This tutorial requires that the Azure SQL database is setup as shown in [Create Azure SQL Database](#Create-Azure-SQL-Database).
+
+- Open your app that you created in 'Set Up Your Local Environment' in VSCode
+- Press 'F1' and search for 'Azure Functions: Create Function'
+- Choose HttpTrigger -> (Provide a function name) -> Company.namespace -> anonymous
+- In the file that opens, replace the 'public static async Task< IActionResult > Run' block with the below code.
+
+    ```csharp
+    public static async Task<IActionResult> Run(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = "employees/{id}")] HttpRequest req,
+        ILogger log,
+        [Sql("select * from Employees where EmployeeId = @EmployeeId",
+        CommandType = System.Data.CommandType.Text,
+        Parameters = "@EmployeeId = {id}",
+        ConnectionStringSetting = "SqlConnectionString")]
+        IEnumerable<Employee> employee)
+    {
+        return new OkObjectResult(employee);
+    }
+    ```
+
+    *In the above, "select from Employees where EmployeeId = @EmployeeId" is the SQL script run by the input binding. The CommandType on the line below specifies whether the first line is a query or a stored procedure. The line below that determines the parameters. On the next line, the ConnectionStringSetting specifies that the app setting that contains the SQL connection string used to connect to the database is "SqlConnectionString." For more information on this, see the [Input Binding Samples](#Input-Binding-Samples) section*
+
+- Add 'using System.Collections.Generic;' to the namespaces list at the top of the page.
+- Currently, there is an error for the Ienumerable. We'll fix this by creating an Employee class.
+- Create a new file and call it 'Employee.cs'
+- Paste the below in the file. These are the column values of our SQL Database table.
+
+    ```csharp
+    namespace Company.Function {
+        public class Employee{
+            public int EmployeeId { get; set; }
+            public string LastName { get; set; }
+            public string FirstName { get; set; }
+            public string Company { get; set; }
+            public string Team { get; set; }
+        }
+    }
+    ```
+
+- Navigate back to your HttpTrigger file. We can ignore the 'Run' warning for now.
+- Open the local.settings.json file, and in the brackets for "Values," verify there is a 'SqlConnectionString.' If not, add it.
+- Hit 'F5' to run your code. Both an HttpTrigger and SQLBinding will run.
+- Click the link that appears in your terminal. If you get a 'HTTP ERROR 500' message, replace the browser url after the last '/' with 1. This is the id of the employee to be outputted. The url should look like 'localhost:7071/api/employees/1'
```

Comment:
lol, in the future we should include a sample that has a happy landing page and not a 500 error, but probably ok for now

Reply:
Haha, very fair. When I was originally putting this in, I wanted to include the parameters in the tutorial. Will see if there's a quick update to make a happy landing. If not, we can leave as is for tomorrow

Reply:
Made the input binding tutorial not take a parameter. Still walks them through the input binding but provides a happy landing page. There are lots of examples with parameters still in 'More Samples'